packages/clay-management-toolbar/src/ActiveState.tsx

```diff
@@ -0,0 +1,248 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayButton from '@clayui/button';
+import ClayForm from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLink from '@clayui/link';
+import Item from './Item';
+import ItemList from './ItemList';
+import React from 'react';
+import {ClayDropDownWithBasicItems} from '@clayui/drop-down';
+import {IItem} from '@clayui/drop-down/src/DropDownWithBasicItems';
+import {IPropsBase} from './types';
+
+interface IActionItems extends IItem {
+	quickAction?: boolean;
+}
+
+interface IProps extends IPropsBase {
+	actionItems?: Array<IActionItems>;
+	clearSelectionURL?: string;
+	onClearSelectionClick?: (
+		e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+	) => void;
+	onQuickActionClick?: (
+		e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+	) => void;
+	onSelectAllClick?: (
+		e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+	) => void;
+	selectAllURL?: string;
+	selectedItems?: number;
+	showSelectAllButton?: boolean;
+	supportsBulkActions?: boolean;
+	totalItems?: number;
+}
+
+const ActiveState: React.FunctionComponent<IProps> = ({
+	actionItems = [],
+	checkboxChecked = false,
+	clearSelectionURL,
+	disabled = false,
+	filterItems,
+	labels = {
+		checkbox: 'Select Items',
+		clearSelection: 'Clear',
+		selectAll: 'Select all',
+	},
+	onCheckboxChange = () => {},
+	onClearSelectionClick = () => {},
+	onInfoButtonClick = () => {},
+	onQuickActionClick = () => {},
+	onSelectAllClick = () => {},
+	selectAllURL,
+	selectedItems,
+	showInfoButton = false,
+	showSearch = false,
+	showSelectAllButton = false,
+	sortingOrder,
+	spritemap,
+	supportsBulkActions = false,
+	titles = {
+		infoButton: 'Info',
+	},
+	totalItems,
+}) => {
+	return (
+		<>
+			<ItemList expand>
+				<Item>
+					<ClayForm.Checkbox
+						checked={checkboxChecked}
+						disabled={disabled}
+						label={
+							!filterItems && !sortingOrder && !showSearch
+								? labels.checkbox
+								: undefined
+						}
+						onChange={onCheckboxChange}
+					/>
+				</Item>
+				<Item>
+					<span className="navbar-text">
+						{selectedItems === totalItems ? (
+							'all-selected'
```

Comment:
We'll need to pass a configurable object of language keys for these sorts of messages. This way we can account for other languages.

Reply:
This applies to any explicit english message as well.

Reply:
Yes, I think I'll add it together with the `labels` API.